code/game/objects/items/plushes.dm

```diff
@@ -187,6 +191,11 @@
 			grenade = null
 		return
 	if(istype(I, /obj/item/grenade))
+		if(unstuffable)
+			to_chat(user, "<span class='warning'>No... you should destroy it now!</span>")
+			sleep(10)
```

Comment:
should probably use `do_after` instead of `sleep`

Reply:
Do_afters are for when the process can be interrupted right? This ball of fur(r)y has no brakes.

Reply:
no do_after has a thing that makes it uninterruptible it's generally better to use `do_after` over `sleep` because byond issues ask kevinz if you want a detailed explination

Reply:
i'm pretty sure the best thing to use here would actually be addtimer(CALLBACK()) and not do_after

Reply:
do after is interruptable when you move, it's fine to use sleep as it is but it's **not** fine to use `spawn()`